scripts/README.md

```diff
@@ -12,11 +12,16 @@ Contains all the scripts relevant to benchmarking `cardano-node`. See the benchm
 #### buildkite
 Contains scripts relevant to IOHK's CI.
 
-#### byron-shelley
-Contains a script that sets up a cluster beginning in the Byron era and can transition to the Shelley era.
-
+#### byron-shelley-allegra-mary
+Contains a script that sets up a cluster beginning in the Byron era and can transition to the Shelley era. You can also start a cluster in the Shelley, Allegra or Mary era by supplying an argument to `mk-files.sh`.
+E.g
+```bash
+./mk-files.sh shelley # Starts nodes in Shelley era
+./mk-files.sh allegra # Starts nodes in Allegra era
+./mk-files.sh mary    # Starts nodes in Mary era
+```
```

Comment:
Seems like it should be run with:

```
./scripts/byron-to-mary/mkfiles.sh mary
```

instead.

Reply:
:+1: I'll make the change